packages/ui5-library-generator/src/app/types.ts

```diff
@@ -0,0 +1,16 @@
+import type { Prompts, AppWizard } from '@sap-devx/yeoman-ui-types';
+import type { UI5LibraryAnswers } from '@sap-ux/ui5-library-inquirer';
+
+export interface LibAnswers {
```

Comment:
We re-defining an existing interface `UI5LibraryAnswers` is the definition that should be used?

Reply:
This was added by mistake and not used. I'll remove